plugin/src/test/com/exadel/aem/toolkit/plugin/maven/WidgetsTest.java

```diff
@@ -13,6 +13,8 @@
  */
 package com.exadel.aem.toolkit.plugin.maven;
 
+import com.exadel.aem.toolkit.test.widget.SliderWidget;
```

Comment:
(cosmetic) Please make sure that the `.editorconfig` file is recognized and honored by your Idea. `com.exadel.*` imports always go last